test/specs/lib/FocusZone-test.tsx

```diff
@@ -0,0 +1,1451 @@
+import * as React from 'react'
+
+import * as ReactDOM from 'react-dom'
+import * as ReactTestUtils from 'react-dom/test-utils'
+import { KeyCodes } from '@uifabric/utilities'
```

Comment:
We already bundle and use [`keyboardKey`](https://www.npmjs.com/package/keyboard-key) which includes future proof `key` names and a `keyCode` util.  Suggest, we stick to that for consistency and future proofing.